ci-cd-bootstrap/params/npm-build-pod/build_params

```diff
@@ -0,0 +1,5 @@
+SOURCE_REPOSITORY_URL=https://github.com/InfoSec812/labs-ci-cd
+SOURCE_CONTEXT_DIR=docker/npm-build-pod
+BUILDER_IMAGE_NAME=registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7:latest
```

Comment:
Hmmmm . . . That would require that we change the params for every deployment... While I'm not against that entirely, I wonder if there is a way to somehow tie this to the version of OpenShift where this is being deployed. Any thoughts?